xml/issue2567.xml

```diff
@@ -86,6 +86,41 @@ BaseCharacteristic is the last type in the list.</del><br/>
 [ Note: This means a specialization of <tt>disjunction</tt> does
 not necessarily <del>have a BaseCharacteristic
 of</del> <ins>derive from</ins> either true_type or
+false_type. — end note ]</p></blockquote>
+</discussion>
+
+<resolution>
+<p>In [meta.logical]/3, edit as follows:</p>
+
+<p>The <del>BaseCharacteristic of a</del> specialization <tt>conjunction&lt;B1, ..., BN&gt;</tt>
+<ins>has a public and unambiguous base that is either<br/>
+* the first type <tt>Bi</tt> in the list <tt>true_type, B1, ..., BN</tt> for
+which <tt>Bi::value == false</tt>, or<br/>
+* if there is no such <tt>Bi</tt>, the last type in the list.</ins><br/>
+<del>is the first type <tt>Bi</tt> in the list <tt>true_type, B1, ..., BN</tt> for
+which <tt>Bi::value == false</tt>, or if every <tt>Bi::value != false</tt>, the
+BaseCharacteristic is the last type in the list.</del><br/>
+<ins>The member names of the base class shall not be hidden and be unambiguously
+available in <tt>conjunction</tt>.</ins><br/>
+[ Note: This means a specialization of <tt>conjunction</tt> does
+not necessarily <del>have a BaseCharacteristic
+of</del> <ins>inherit from</ins> either true_type or
+false_type. — end note ]</p>
+
+<p>In [meta.logical]/6, edit as follows:</p>
+<p>The <del>BaseCharacteristic of a</del> specialization <tt>disjunction&lt;B1, ..., BN&gt;</tt>
+<ins>has a public and unambiguous base that is either<br/>
+* the first type <tt>Bi</tt> in the list <tt>true_type, B1, ..., BN</tt> for
+which <tt>Bi::value != false</tt>, or<br/>
+* if there is no such <tt>Bi</tt>, the last type in the list.</ins><br/>
+<del>is the first type <tt>Bi</tt> in the list <tt>true_type, B1, ..., BN</tt> for
+which <tt>Bi::value != false</tt>, or if every <tt>Bi::value == false</tt>, the
+BaseCharacteristic is the last type in the list.</del><br/>
+<ins>The member names of the base class shall not be hidden and be unambiguously
```

Comment:
@timsong-cpp We discussed operator= in the room and remarked that this is no different than any of the other type traits already using the BaseCharacteristic language. We didn't discuss the injected class name; I'll make sure that's brought up again when discussing this.

Consensus was to have a separate issue on the "names that must be hidden" issue.

I'll look into providing wording here which also deals with the names that must be hidden, stand by.